addons/crm/views/crm_lead_views.xml

```diff
@@ -1044,6 +1058,19 @@ if record:
             <field name="act_window_id" ref="crm_lead_action_pipeline"/>
         </record>
 
+        <record id="crm_lead_duplicated_leads" model="ir.actions.act_window">
```

Comment:
Guidelines: crm_lead_action_duplicates or something like that (model - action - details)